src/index.js

```diff
@@ -81,6 +83,7 @@ export default class Chart extends PureComponent<Props> {
         source={source}
         originWhitelist={["*"]}
         onMessage={this.onMessage}
+        onContentProcessDidTerminate={this.onContentProcessDidTerminate}
```

Comment:
❌ 
```suggestion
        onContentProcessDidTerminate={this.chart.current.reload}
```

Reply:
@bogdanvidican it doesn't work if change to this